.github/workflows/test-full-sync.yml

```
@@ -43,113 +43,35 @@ on:
-          # generate Docker tags based on the following events/attributes
-          tags: |
-            type=schedule
-            type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
-            type=sha
-
-      # Setup Docker Buildx to allow use of docker cache layers from GH
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v1
-
-      - name: Authenticate to Google Cloud
-        id: auth
-        uses: google-github-actions/auth@v0.7.1
-        with:
-          workload_identity_provider: 'projects/143793276228/locations/global/workloadIdentityPools/github-actions/providers/github-oidc'
-          service_account: 'github-service-account@zealous-zebra.iam.gserviceaccount.com'
-          token_format: 'access_token'
-
-      - name: Login to Google Artifact Registry
-        uses: docker/login-action@v1.14.1
-        with:
-          registry: us-docker.pkg.dev
-          username: oauth2accesstoken
-          password: ${{ steps.auth.outputs.access_token }}
-
-      - name: Login to Google Container Registry
-        uses: docker/login-action@v1.14.1
-        with:
-          registry: gcr.io
-          username: oauth2accesstoken
-          password: ${{ steps.auth.outputs.access_token }}
-
-      # Build and push image to Google Artifact Registry
-      - name: Build & push
-        id: docker_build
-        uses: docker/build-push-action@v2.10.0
-        with:
-          target: tester
-          context: .
-          file: ./docker/Dockerfile
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          build-args: |
-            NETWORK=${{ github.event.inputs.network || env.NETWORK }}
-            SHORT_SHA=${{ env.GITHUB_SHA_SHORT }}
-            RUST_BACKTRACE=${{ env.RUST_BACKTRACE }}
-            RUST_LIB_BACKTRACE=${{ env.RUST_LIB_BACKTRACE }}
-            COLORBT_SHOW_HIDDEN=${{ env.COLORBT_SHOW_HIDDEN }}
-            ZEBRA_SKIP_NETWORK_TESTS="1"
-            CHECKPOINT_SYNC=${{ github.event.inputs.checkpoint_sync || true }}
-            RUST_LOG=debug
-            SENTRY_DSN=${{ secrets.SENTRY_ENDPOINT }}
-          push: true
-          cache-from: type=registry,ref=${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:${{ env.GITHUB_REF_SLUG_URL }}-buildcache
-          cache-to: type=registry,ref=${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:${{ env.GITHUB_REF_SLUG_URL }}-buildcache,mode=max
+    uses: ./.github/workflows/docker-image-build.yml
+    with:
+      dockerfile_path: ./docker/Dockerfile
+      dockerfile_target: tester
+      image_name: zebrad-test
+      network: Mainnet
+      checkpoint_sync: true
+      rust_backtrace: full
+      rust_lib_backtrace: full
+      colorbt_show_hidden: '1'
+      zebra_skip_ipv6_tests: '1'
+      rust_log: debug
```

Performance: debug logging slows down Zebra a lot. Let's only turn it on if we really need it. I'll open a separate PR for this change, because #4155 is urgent.

```suggestion
      rust_log: info
```